PACKAGE.md

```diff
@@ -86,17 +86,16 @@ It should typically be enough to run `osc ar` to add new and delete removed file
 osc commit
 ```
 
-## Step 7. Submit Requests to OpenSUSE Factory and IBS
+## Step 7. Submit Requests to OpenSUSE Factory and SLES
 
 To get a maintenance request accepted, each changelog entry needs to reference a bug or feature
-request with `bnc#123` or `fate#123`.
+request with `bsc#123` or `fate#123`.
 
 ### Factory First
 
 To submit a request to openSUSE Factory, issue this commands in the console:
 
 ```bash
-cd package
```

Comment:
Is that not correct anymore?

Reply:
just not needed, you can call that osc command from anywhere.

Reply:
:thinking: I don't want accidentally submit something while testing this, but doesn't `osc` expect the `.changes` and `.spec` file and tarball of the package name you tell it (`SUSEConnect` in that case) in the current working directory?

If you are sure it does not, go ahead. Worst-case would be that add that line back later when we find out it was needed :wink: